glusterd2/events/nfs-ganesha.go

```diff
@@ -0,0 +1,37 @@
+package events
+
+import (
+	log "github.com/sirupsen/logrus"
```

Comment:
The general convention followed for imports is as follows:
```
standard golang packages

gd2 packages

third-party packages
```

In this case:
```go
import (
    "fmt"
    "os/exec"

    log "github.com/sirupsen/logrus"
)
```

Reply:
Done